NGS2_WITNESS_Cycle1_experiment1_prep.R

```diff
@@ -0,0 +1,422 @@
+# Created by Matt Hoover (matt_hoover@gallup.com) for Gallup in September 2017
+# determine system environment
+if(Sys.info()['sysname'] == 'Darwin') {
+    rm(list=ls())
+
+    # Install packages that are required for this file
+    list.of.packages <- c("dplyr", "pacman", "reshape2")
+    new.packages <- list.of.packages[!(list.of.packages %in%
+                                       installed.packages()[,"Package"])]
+    if(length(new.packages)) install.packages(new.packages)
+    lapply(list.of.packages, require, character.only = TRUE)
+
+    pacman::p_load(multiwayvcov, lmtest)
+}
```

Comment:
I'd rather not. I don't `source()` the master script, so pulling in libraries doesn't happen for me there. Given the `if(Sys.info()['sysname'] == 'Darwin') {` we should be fine -- that'll only execute this block if someone is using a Mac, so anyone running from a Gallup server will skip over this entirely.

Reply:
Hmmm, I'm betraying my PC-centrism here, but... is there a way to pull all the libraries from a central script, rather than having library calls on each separate script?

Reply:
Yeah, the way you have it in `NGS2_WITNESS_Cycle1_MASTER.R` is a great way to do it... but I don't run the entire pipeline each time I process the Breadboard data.

Reply:
How about I pull the library calls into a separate script that does only that?

Reply:
We're bikeshedding at this point. However you want it handled will be fine; I'll adjust accordingly. Lay out your plan here and I'll adjust the PR.

Reply:
OK, let us ignore this for now.